### ISSUE TITLE
Error when including a namspace without an alias in a component

### ISSUE THREAD

**Issue by jmayaalv** [polyfy/polylith#67]
**Describe the bug**
When a component includes a namespace without an alias the `poly info` raises the error 
`Don't know how to create ISeq from: clojure.lang.Symbol`

**To Reproduce**
1. Create a component
2. in the interface of the component include a namespace without an alias ex:
```
(ns edge.money.interface
  (:require [com.fulcrologic.guardrails.core :refer [>def >defn =>]]
            [edge.money.core :as core]
            [edge.money :as money]
            edge.money.reader))
```
Problem is solved when changing
`edge.money.reader` for `edge.money.reader :as reader` 
2. Run poly info
```
clj -M:poly info
```
3. See error Don't know how to create ISeq from: clojure.lang.Symbol

**Expected behavior**
An alias for the namespace is not mandatory. The use case is common to force the namespace to be loaded so a namespace without an alias shouldn't break the polylith tool.

**Workspace Attachment**
If applicable, add output of following command as attachment:
- `poly ws out:ws.edn`
Same error appear when running this command

**Operating System (please complete the following information):**
 - OS: MacOs
 - Version: BigSur

**Versions (please complete the following information):**
 - Java: "1.8.0_192-ea"
 - Poly: "0.1.0-alpha9"

**Comment by furkan3ayraktar** [comment]
Hi @jmayaalv,

Thanks for reporting this issue. The problem will be solved if you could alter your require statement to this instead:

```clojure
(ns edge.money.interface
  (:require [com.fulcrologic.guardrails.core :refer [>def >defn =>]]
            [edge.money.core :as core]
            [edge.money :as money]
            [edge.money.reader]))
```

The change I made is that instead of `edge.money.reader` you have `[edge.money.reader]` and everything works without any issues. 

As you can read more in detail [here](https://stuartsierra.com/2016/clojure-how-to-ns.html#vectors-or-lists-1), a best practice when it comes to declaring namespaces is to use vectors in the `:require` statements. Polylith does static code analysis in order to understand the relations between components, bases, and external libraries. In order to achieve this analysis, we took the common practices when it comes to declaring namespaces, variables, and functions. It is not possible to support all different ways of coding styles but we believe we picked the most generally accepted ones.

Let me know if you have any more questions around the topic.
Furkan

**Comment by jmayaalv** [comment]
Thanks @furkan3ayraktar :) and here i was for 3 years thinking it was a good practice not to enclose it on a vector. 

That fixes the problem, however it was quiet painful to find the problem as the only error reported was quiet cryptic and had to manually start commenting pieces of code until managed to reproduce it. So a good  error message would be very useful but I understand this could be difficult to achive, so maybe a warning  in the readme could help other people specially migrating existing apps to polylith?

**Comment by furkan3ayraktar** [comment]
I agree, we can improve the errors especially when we are parsing the code. At least, we should be able to direct users to a namespace where the parsing error occurred. 

I also believe it could be a good addition to the README where we explain how Polylith expects the `:require` section of a namespace to be written.

**Comment by futuro** [comment]
As a note to the future, this issue should be resolved by PR #74.

**Comment by tengstrand** [comment]
Fixed.
